projects/demo/src/modules/experimental/block-details/examples/1/index.less

```diff
@@ -0,0 +1,3 @@
+:host {
+    width: 15rem;
```

Comment:
Is it really required ?

<img width="1491" alt="width" src="https://github.com/taiga-family/taiga-ui/assets/35179038/8e6b600a-1018-4e8f-a994-f3ed7523744b">